clang/test/CodeGen/cheri/vaddr-mode.c

```diff
@@ -229,12 +229,33 @@ __int128 uintcap_to_oversized(__uintcap_t cap) {
   return (__int128)cap;
 }
 
-// Casts from a capability to an oversized integer previously crashed Clang,
```

Comment:
I think it's fine to keep the comment here since __intcap is a capability.

Reply:
Sorry commented on the overall diff - I would move this comment up or move the intcap case down.

Reply:
The comment is here because __intcap wasn't crashing; see https://github.com/CTSRD-CHERI/llvm-project/commit/37a80dc5dcdc411757c646787dbe0ee7116f044a